src/core/Akka.MultiNodeTestRunner.Shared/Sinks/ConsoleMessageSinkActor.cs

```diff
@@ -151,17 +151,17 @@ private void WriteSpecMessage(string message)
             Console.ResetColor();
         }
 
-        private void WriteSpecPass(int nodeIndex, string message)
+        private void WriteSpecPass(int nodeIndex, string nodeRole, string message)
         {
             Console.ForegroundColor = ConsoleColor.Green;
-            Console.WriteLine("[NODE{0}][{1}]: SPEC PASSED: {2}", nodeIndex, DateTime.UtcNow.ToShortTimeString(), message);
+            Console.WriteLine("[NODE{0}:{1}][{2}]: SPEC PASSED: {3}", nodeIndex, nodeRole, DateTime.UtcNow.ToShortTimeString(), message);
             Console.ResetColor();
         }
 
-        private void WriteSpecFail(int nodeIndex, string message)
+        private void WriteSpecFail(int nodeIndex, string nodeRole, string message)
         {
             Console.ForegroundColor = ConsoleColor.Red;
-            Console.WriteLine("[NODE{0}][{1}]: SPEC FAILED: {2}", nodeIndex, DateTime.UtcNow.ToShortTimeString(), message);
+            Console.WriteLine("[NODE{0}:{1}][{2}]: SPEC FAILED: {3}", nodeIndex, nodeRole, DateTime.UtcNow.ToShortTimeString(), message);
```

Comment:
Love this change. Having the rolename listed in the log will make things so much easier.